python/pfs/drp/stella/utils.py

```diff
@@ -128,3 +131,60 @@ def createPfsFiberTrace(dataId, fiberTraceSet, nRows):
         pfsFiberTrace.profiles.append(profOut)
 
     return pfsFiberTrace
+
+def readWavelengthFile(wLenFile):
+    # read wavelength file
+    hdulist = pyfits.open(wLenFile)
+    tbdata = hdulist[1].data
+    traceIdsTemp = np.ndarray(shape=(len(tbdata)), dtype='int')
+    xCenters = np.ndarray(shape=(len(tbdata)), dtype='float32')
+    yCenters = np.ndarray(shape=(len(tbdata)), dtype='float32')
+    wavelengths = np.ndarray(shape=(len(tbdata)), dtype='float32')
+    traceIdsTemp[:] = tbdata[:]['fiberNum']
+    traceIds = traceIdsTemp.astype('int32')
+    wavelengths[:] = tbdata[:]['pixelWave']
+    xCenters[:] = tbdata[:]['xc']
+    yCenters[:] = tbdata[:]['yc']
+    return {'xCenters': xCenters, 'yCenters': yCenters, 'wavelengths': wavelengths, 'traceIds': traceIds}
```

Comment:
Why are you returning a `dict` here? I think any of `pipeBase.struct`, `collections.NamedTuple` or just a plain old `tuple` would be more idiomatic.